build.gradle

```diff
@@ -242,6 +242,17 @@ dependencyManagement {
         dependencySet(group: 'commons-beanutils', version: '1.9.4') {
             entry 'commons-beanutils'
         }
+
+        // solves CVE-2020-5407
```

Comment:
Hi David,
we have suppression on that one:
```
  <suppress until="2030-01-01">
    <notes><![CDATA[
     Suppressing SAML vulnerability as it is not used in this project (see: https://nvd.nist.gov/vuln/detail/CVE-2020-5407)
   ]]></notes>
    <gav regex="true">org.springframework.security:spring-security-*.*</gav>
    <cpe>cpe:2.3:a:security-framework_project:security-framework:5.2.1:*:*:*:*:*:*:*</cpe>
    <cve>CVE-2020-5407</cve>
  </suppress>
```
we do not need changes in build.gradle